easybuild/easyconfigs/f/FRUIT/fruit_3.4.3-ifort-external.patch

```diff
@@ -0,0 +1,41 @@
+diff -Nru fruit_3.4.3-orig/rake_base.rb fruit_3.4.3/rake_base.rb
```

Comment:
@SimonPinches Please include a description of this patch above this line, and mention the author or include a reference to where this patch came from (keep in mind that this implies that the patch checksum will be need to be updated)

Reply:
@boegel Done, I've also split this patch into two since it covered two elements.